net-proxy/squid/squid-5.4.1.ebuild

```diff
@@ -52,8 +52,8 @@ COMMON_DEPEND="acct-group/squid
 	esi? ( dev-libs/expat dev-libs/libxml2 )
 	gnutls? ( >=net-libs/gnutls-3.1.5:= )
 	logrotate? ( app-admin/logrotate )
-	>=sys-libs/db-4:*
```

Comment:
Revbump as it'll be automagic / will have worked if people had the right stuff installed accidentally.

(just git mv)

Reply:
Ok, pushed.